docs/topics/readme/osio-rest-http-secured-README.adoc

```diff
@@ -21,15 +21,15 @@ NOTE: Deploying your booster to Stage may take a few minutes to complete.
 NOTE: This will create a popup window so you may have to enable popups in your browser.
 
 * Make a change to your booster (e.g. edit `${fileLocation}`).
-* Click the _Run_ button (the triangle in the top next to the _Debug_ button) and choose _Run_ to build and deploy your booster in a temporary Che environment. Use the _Preview_ URL at the top of the log window. Observe that your changes appear.
+* Click the _Run_ button (the triangle in the top next to the _Debug_ button) and choose _Run_ to build and deploy your booster in a temporary Che environment. Use the _Preview_ URL at the top of the log window. Verify that your changes took effect.
 
 == Promoting the Changes
 * Click _Git_ -> _Add to Index_ to stage your change.
 * Click _Git_ -> _Commit_ to commit your change. Ensure _Push to Origin_ is checked before clicking _Commit_.
-* Return to your workspace and observe a new pipeline has started. Once complete click on _Stage - 1.0.2_ under _Applications_ and observe your changes.
-* Click on _Run - 1.0.1_ under _Applications_ and observe that your changes *DO NOT* appear.
+* Return to your workspace and verify a new pipeline has started. Once complete click on _Stage - 1.0.2_ under _Applications_ and verify your changes.
```

Comment:
...verify that a new pipeline started...

Reply:
Will fix.